architecture/additional_concepts/flannel.adoc

```diff
@@ -29,4 +31,9 @@ responsibile for:
 
 Each *flanneld* agent provides this infomation to a centralized *etcd* store so
 other agents on hosts can create an overlay network and route packets to
-any container within the *flannel* network.
\ No newline at end of file
+any container within the *flannel* network.
+
+The following diagram illustrates the architecture and data flow from one
+container to another using a *flannel* network:
+
+image::flannel.png[Flannel Communication]
```

Comment:
Hm... I was thinking host-gw didn't use veths or docker0, but maybe that's wrong. But I'm pretty sure there shouldn't be any "flannel0" at least. Anyway, I guess this means you need to find someone who knows more about flannel than me to review it.